test/libsolidity/util/TestFileParser.cpp

```diff
@@ -95,11 +95,19 @@ vector<dev::solidity::test::FunctionCall> TestFileParser::parseFunctionCalls(siz
 						m_lineNumber++;
 					}
 
-					expect(Token::Arrow);
-					call.expectations = parseFunctionCallExpectations();
+					if (accept(Token::Arrow, true))
+					{
+						call.omitsArrow = false;
+						call.expectations = parseFunctionCallExpectations();
+						if (accept(Token::Newline, true))
+							m_lineNumber++;
+					}
+					else
+					{
+						call.expectations.failure = false;
+						call.displayMode = FunctionCall::DisplayMode::SingleLine;
```

Comment:
Why is this needed? Could the arguments be given in a different line?

Reply:
Right, need to check that particular case.

Reply:
Oh, reading this again, I might have misunderstood your comment. Yes, you can define a test like below. This will set the display mode to multi-line, which will have a different formatting then single-line:
```
contract C {
        function f(uint a, uint b, uint c, uint d, uint e) public returns (uint) {
            return a + b + c + d + e;
        }
    }
// ----
// f(uint256,uint256,uint256,uint256,uint256):
// 1,
// 1,
// 1,
// 1,
// 1
```
The above would be still single-line, since the "newlined" arguments are not affecting the display mode. Only having the arrow on a newline would change the mode. Since this is now the shorthand version without an arrow, the default should be single-line again:
```
Obtained result:
  // f(uint256,uint256,uint256,uint256,uint256):
  // 1,
  // 1,
  // 1,
  // 1,
  // 1 -> 5
```

Reply:
Ok!